clang/test/Sema/ext_vector_casts.c

```diff
@@ -51,6 +53,9 @@ static void test(void) {
     ivec4 -= ivec4;
     ivec4 |= ivec4;
     ivec4 += ptr; // expected-error {{cannot convert between vector and non-scalar values ('int4' (vector of 4 'int' values) and 'int *')}}
+
+    bvec4 != 0; // expected-warning {{inequality comparison result unused}} \
```

Comment:
Isn't the vector element type here `float` and not `bool`? (I'm trying to understand how the test case relates to the changes and feel like I must be missing something; perhaps the bool vector is from the result of the inequality operator?)

Reply:
@AaronBallman Thanks for the review, and I'm sorry for the confusion. There was a typo — the type should be `_Bool`. I've changed it to the appropriate type now.